coalib/output/ConsoleInteraction.py

```diff
@@ -561,8 +561,12 @@ def choose_action(console_printer, actions):
 
         try:
             line = format_lines("Please enter the number of the action "
-                                "you want to execute (Ctrl-D to exit). ")
-            choice = int(input(line))
+                                "you want to execute (Ctrl-D to exit). "
+                                "Default value is 0")
```

Comment:
idea! Sorry but I think the note should rather go in line 556 like `(Default)` or so